cron/builder/main.go

```diff
@@ -15,17 +16,17 @@ import (
 	"github.com/codeship/codeship-go"
 )
 
-// ParamOrganization is the environment variable for
-// the Codeship organization
-const ParamOrganization = "organization"
-
-// ParamPassword is the environment variable for
+// EnvKeyCSPassword is the environment variable for
 // the Codeship password
-const ParamPassword = "password"
+const EnvKeyCSPassword = "CS_PASSWORD"
 
-// ParamUsername is the environment variable for
+// EnvKeyCSUsername is the environment variable for
 // the Codeship username
-const ParamUsername = "username"
+const EnvKeyCSUsername = "CS_USERNAME"
+
+// ParamOrganization is the environment variable for
+// the Codeship organization
+const ParamOrganization = "organization"
```

Comment:
Would it be worth naming this something like `EnvKeyOrganization` to match the naming pattern of the other two environment variable name `const`s right above this? (Or maybe it's just the comment that is not correct...)

Reply:
The comment is not correct. It was an environment variable before it was moved into SSM. I'll correct the comment.